Resources/string.csv

```diff
@@ -378,55 +378,55 @@ Command.h_args,Available Args (clipped),使用可能な引数 (略称),可用参
 CantUse.lastroles,Can't use /lastroles in the game.,試合中に/lastrolesを使用することはできません。,不可以在游戏中使用/lastroles命令,不可以在遊戲中使用/lastroles指令
 
 ## メッセージ
-Message.SetToSeconds,Set to {0} seconds.,{0}秒に設定されました,信息发送冷却时间为{0}秒,訊息發送冷卻已設定為{0}秒
-Message.MessageWaitHelp,Specify the first argument in seconds.,第一引数を秒数で指定します。,指定一个数设定秒数,在指令後方指定一個數字來設定秒數
-Message.TemplateNotFoundHost,"No messages matching "{0}" were found.\nPlease add to template.txt like.\n{0}:contents\n\nDefined Tags:\n{1}",「{0}」に該当するメッセージが見つかりませんでした。\n{0}:内容\nのようにtemplate.txtに追記してください。\n\n定義されているタグ:\n{1},未找到与“{0}”对应的消息。 \n{0}: 内容\n 请添加到 template.txt。 \n\n标签定义：\n{1},"沒有找到與「{0}」相關的訊息,\n請將內容加入template.txt作為{0}的定義內容。\n\n定義標籤:\n{1}"
-Message.TemplateNotFoundClient,"No messages matching "{0}" were found.",「{0}」に該当するメッセージが見つかりませんでした。,未找到与“{0}”相关的消息。,沒有找到與「{0}」相關的訊息
-Message.SyncButtonLeft,The emergency meeting button can be used {0} more times,緊急会議ボタンはあと{0}回使用できます,您可以多次使用紧急会议按钮 {0},"全場拍桌數量還剩下{0}次,請謹慎拍桌"
-Message.Executed,{0} was executed,{0}を処刑しました,,{0}被處刑了
-Message.HideGameSettings,Game settings are hidden by the host.,ゲーム設定はホストによって秘匿されています。,游戏设置被房主隐藏。,遊戲設置被房主隱藏。
+Message.SetToSeconds,Set to {0} seconds.,{0}秒に設定されました,该参数已设置为{0}秒。,訊息發送冷卻已設定為{0}秒
+Message.MessageWaitHelp,Specify the first argument in seconds.,第一引数を秒数で指定します。,请以秒为单位指定第一个参数。,在指令後方指定一個數字來設定秒數
+Message.TemplateNotFoundHost,No messages matching {0}" were found.\nPlease add to template.txt like.\n{0}:contents\n\nDefined Tags:\n{1}",「{0}」に該当するメッセージが見つかりませんでした。\n{0}:内容\nのようにtemplate.txtに追記してください。\n\n定義されているタグ:\n{1},找不到与「{0}」匹配的信息。\n请将需要的信息以如下格式加入template.txt：\n{0}（内容）：{1}（定义标签）,沒有找到與「{0}」相關的訊息,\n請將內容加入template.txt作為{0}的定義內容。\n\n定義標籤:\n{1}
+Message.TemplateNotFoundClient,No messages matching {0}" were found.",「{0}」に該当するメッセージが見つかりませんでした。,找不到与「{0}」匹配的信息。,沒有找到與「{0}」相關的訊息
+Message.SyncButtonLeft,The emergency meeting button can be used {0} more times,緊急会議ボタンはあと{0}回使用できます,还可以召开{0}次紧急会议,全場拍桌數量還剩下{0}次,請謹慎拍桌
```

Comment:
CSV must be enclosed in double quotation marks such as "xxxxxxxxx,yyyyyyy" when a comma is included in the text.